### DISCUSSION TITLE
Scan a network and import connection automatically

### DISCUSSION THREAD

**Discussion by c0r3dump3d** [electerm/electerm#3528] [Ideas]
Hi, very nice software! Thank you very much for your work. As a idea and as a sysadmin I need to import thousand of host from a network scanning, it would be very interesting if you could scan a network and automatically import the active hosts, or import from an nmap scan.

**Comment by zxdong262** [comment]
In bookmark manage page, could export/import bookmarks, so if you can scan the network and generate a formatted json, then could just import these networks as bookmarks, does this works for you?

**Reply by c0r3dump3d** [reply]
Hi, thanks for the answer, I will try!!
